dogs.html

```diff
@@ -0,0 +1,167 @@
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+       
+            <div class="column">
+                <div class="card">
+                  <img src="images/duffy-card.jpg" alt="Picture of Duffy">
+                  <h3>Duffy</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+        
+            <div class="column">
+                <div class="card">
+                  <img src="images/lucas-card.jpg" alt="Picture of Lucas">
+                  <h3>Lucas</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+        
+            <div class="column">
+                <div class="card">
+                  <img src="images/jake-card.jpg" alt="Picture of Jake">
+                  <h3>Jake</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+        
+            <div class="column">
+                <div class="card">
+                  <img src="images/angus-card.jpg" alt="Picture of Angus">
+                  <h3>Angus</h3>
+                  <h4>Cost to adopt: $123.45</h4>
+                  <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                  <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+           
+            <div class="column">
+                <div class="card">
+                <img src="images/piper-card.jpg" alt="Picture of Piper">
+                <h3>Piper</h3>
+                <h4>Cost to adopt: $123.45</h4>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <a href="#" class="card-button">Adopt</a>
+                </div>
+            </div>
+    
+            <div class="column">
+                <div class="card">
+                <img src="images/maximus-card.jpg" alt="Picture of Maximus">
+                <h3>Maximus</h3>
+                <h4>Cost to adopt: $123.45</h4>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <a href="#" class="card-button">Adopt</a>
```

Comment:
Tip. These will be buttons that won't navigate anywhere, but will be used for simple javascript.